tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py

```diff
@@ -181,62 +177,58 @@ def test_node_disconnected(self):
 async def test_connect_attempts():
     """Test connect attempts."""
     # test connects
-    with tempfile.TemporaryDirectory() as dirname:
-        con = _make_libp2p_client_connection(
-            data_dir=dirname, peer_public_key=make_crypto(DEFAULT_LEDGER).public_key
-        )
```

Comment:
all directories in tests are dynamically created as part of the base class, or the respective and libp2p connection that is created using the helper functions `make_libp2p_connection*`, as sub directories of a temporary master directory to ensure all files are deleted